x-pack/plugins/security_solution/common/endpoint/types/trusted_apps.ts

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { TypeOf } from '@kbn/config-schema';
+import { GetTrustedAppsRequestSchema } from '../schema/trusted_apps';
+import { ExceptionListItemSchema } from '../../../../lists/common';
+import { FoundExceptionListItemSchema } from '../../../../lists/common/schemas/response';
+
+/** API request params for retrieving a list of Trusted Apps */
+export type GetTrustedAppsListRequest = TypeOf<typeof GetTrustedAppsRequestSchema.query>;
+export type GetTrustedListAppsResponse = Pick<
+  FoundExceptionListItemSchema,
+  'per_page' | 'page' | 'total'
+> & {
+  data: TrustedApp[];
+};
+
+/** Type for a new Trusted App Entry */
+export type NewTrustedApp = Pick<
+  ExceptionListItemSchema,
+  'created_at' | 'created_by' | 'name' | 'description' | 'entries'
```

Comment:
> NewTrustedApp should not have created_at and created_by, this should not be controllable (or give an illusion that we allow to control it) through API as it would be a vulnerability.

I think this is just the return value for the Trusted App which would include `create_by`, etc.  The interface for Create request wouldn't include those.  So maybe just a rename here.

@paul-tavares let me know your thoughts